.github/workflows/ci_workflows.yml

```diff
@@ -27,7 +27,7 @@ jobs:
     steps:
     - name: Check base branch
       uses: actions/github-script@60a0d83039c74a4aee543508d2ffcb1c3799cdea  # v7.0.1
-      if: github.event_name == 'pull_request'
+      if: github.event_name == 'pull_request' && github.repository == 'astropy/astropy'
```

Comment:
I don't see the point of this one. Are you expecting PR to your fork and your fork does not have main?

Reply:
I still don't think it is necessary. It should work as long as your fork default branch is `main`. If not, there is a label you can add to your fork to disable this check.

Reply:
I think that label only exists on Astropy and needs to be made on forks.
Either way, this is automatic and 41 characters and restricts it to Astropy, to which it only applies.